.github/workflows/ci.yml

```diff
@@ -237,14 +237,14 @@ jobs:
           source .tinyenv
           echo ${{ steps.regression_labels.outputs.labels }}
           REGRESSION_LABELS=$(echo "${{ steps.regression_labels.outputs.labels }}" | awk -F, '{for (i=1; i<=NF; i++) if ($i ~ /^--/) print $i}' ORS=',' | sed 's/,$//')
-          echo ${REGRESSION_LABELS}
+          echo "Regression labels: ${REGRESSION_LABELS}"
 
           CONFIG_FILE=./tests/regression.yaml
           BASE_CMD="tb branch regression-tests"
           LABELS_CMD="$(echo ${REGRESSION_LABELS} | tr , ' ')"
           if [ -f ${CONFIG_FILE} ]; then
-              echo "Config file found: ${CONFIG_FILE}"
-              ${BASE_CMD} -f ${CONFIG_FILE} --wait ${LABELS_CMD}
+              echo "Config file '${CONFIG_FILE}' found, skipping regression labels if present"
+              ${BASE_CMD} -f ${CONFIG_FILE} --wait
```

Comment:
we need to pass at least --skip-regression tests 🙏  otherwise you are on a dead end.

Reply:
Ok, in that case, let's improve the documentation, which in my opinion, is key.